### PR TITLE
fixed review comments

### PR DESCRIPTION
**Pull request by anandsinghms**
## Description
</br>
 Brief description of the requested change.

## Checklist
- [x] I have read the instructions mentioned in [Contribute to Code](/Contributing.md).
- [x] I have read and understood the criteria described under [submitting changes](/Contributing.md#submitting-changes). 
- [x] The title of the PR clearly describes the intent of the PR.
- [x] This PR does not introduce any breaking changes to the code.

### CODE REVIEW

File: src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.ServiceConnection.ps1 [hunk 1]
```diff
@@ -336,53 +336,46 @@ class ServiceConnection: ADOSVTBase
...
-                $controlResult.AddMessage([VerificationResult]::Manual,"List of restricted global groups for service connection is not defined in your organization policy. Please update your ControlSettings.json as per the latest AzSK.ADO PowerShell module.");
-            }
+                $controlResult.AddMessage([VerificationResult]::Passed,"No global groups have access to service connection.");
+            }                   
+
+            $restrictedGroups = $null;
+            $restrictedGlobalGroupsForSerConn = $null;
         }
         catch {
             $failMsg = $_
```
Comment (Garima-msft) [inline src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.ServiceConnection.ps1:376]: control should error out in catch block
No need of $failMsg, can directly use $_ in catch block

Comment (anandsinghms) [reply src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.ServiceConnection.ps1]: done

File: src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.ServiceConnection.ps1 [hunk 2]
```diff
@@ -336,53 +336,45 @@ class ServiceConnection: ADOSVTBase
...
+            }                   
 
-        if(![string]::IsNullOrEmpty($failMsg))
-        {
-            $controlResult.AddMessage([VerificationResult]::Manual,"Unable to fetch service connections details. $($failMsg)Please verify from portal that you are not granting global security groups access to service connections");
+            $restrictedGroups = $null;
+            $restrictedGlobalGroupsForSerConn = $null;
         }
+        catch {            
+            $controlResult.LogException($_)
```
Comment (Garima-msft) [inline src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.ServiceConnection.ps1:376]: Add control result error in catch block  

$controlResult.AddMessage([VerificationResult]::Error,"Unable to fetch service connections details.")

File: src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.ServiceConnection.ps1 [hunk 3]
```diff
@@ -336,53 +336,45 @@ class ServiceConnection: ADOSVTBase
     hidden [ControlResult] CheckGlobalGroupsAddedToServiceConnections ([ControlResult] $controlResult)
 	{
         # Any identity other than teams identity needs to be verified manually as it's details cannot be retrived using API
+        $controlResult.VerificationResult = [VerificationResult]::Failed
         $failMsg = $null
```
Comment (Garima-msft) [inline src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.ServiceConnection.ps1:340]: variable is not being used in method, can be removed

**Review by Garima-msft — APPROVED** [review]

**Review by v-soukum — APPROVED** [review]
looks good

**Pull request merged**